boards/cubepilot/cubeorange/nuttx-config/console/defconfig

```diff
@@ -159,12 +158,11 @@ CONFIG_STM32H7_ADC1=y
 CONFIG_STM32H7_ADC3=y
 CONFIG_STM32H7_BBSRAM=y
 CONFIG_STM32H7_BBSRAM_FILES=5
+CONFIG_STM32H7_BDMA=y
```

Comment:
Are you trying to normalizes all the boards regardless of the HW on them? 
SPI6 is not enabled. BDMA is not used and will just waste space in FLASH (it is static bound and referenced in the DMA driver)

Reply:
fixed